modules/ROOT/pages/depl-examples/minimal-bare-metal.adoc

```diff
@@ -57,16 +57,22 @@ For those who want to skip reading the full document, use this summary of comman
 
 IMPORTANT: With this approach, the system you install Infinite Scale on *must* have a GUI present. A headless system has different requirements and needs an extended setup, see xref:accessing-infinite-scale-other-than-localhost[Accessing Infinite Scale Other Than Localhost].
 
-Define a stable binary to use as replacement for `<file_url>` via {ocis-downloadpage-url}/{download-type}/?sort=time&order=desc[download.owncloud.com,window=_blank].
+Define a stable binary to use as replacement for `<file_url>` via {ocis-downloadpage-url}/{download-type}/?sort=time&order=desc[download.owncloud.com,window=_blank], replace `<version>` in the command accordingly.
 
 [source,bash,subs="attributes+"]
 ----
-sudo wget -O {ocis_bin}/ocis <file_url>
+sudo wget -P {ocis_bin} <file_url>
```

Comment:
This assumes, that <file_url> ends in .../ocis-<version> -- not sure how to best mention this important detail.

Reply:
If one cant distingusih the full url and the filename, he should not do installing at all... We can assume he knows.

Reply:
file_url=https://download.owncloud.com/ocis/ocis/testing/7.0.0-rc.2/ocis-7.0.0-rc.2-linux-amd64
version=7.0.0-rc.2

seems legit, but won't work.

version=7.0.0-rc.2-linux-amd64 is needed here, which does not look like a version. 

Maybe we should change the wget command to:

`sudo wget -O {ocis_bin}/ocis-<version> <file_url>`

Reply:
we have `<file_url>` this is fully correct.
for `version`, I would then suggest to use `file_name` instead.

this make it very clear, because it is only the last segment of the url.
I write a commit with the change.

Reply:
commit added